taipy/core/_entity/_reload.py

```diff
@@ -104,6 +106,4 @@ def _get_manager(manager: str) -> _Manager:
         "job": _JobManagerFactory._build_manager(),
         "task": _TaskManagerFactory._build_manager(),
         "submission": _SubmissionManagerFactory._build_manager(),
-    }[
-        manager
-    ]  # type: ignore
+    }[manager]  # type: ignore
```

Comment:
WHy do we need to add a "# type: ignore" here?

Reply:
apparently we don't need it!